nucliadb_node/src/shards/providers/provider_traits.rs

```diff
@@ -55,17 +46,3 @@ pub trait ShardWriterProvider {
 
     fn get_metadata(&self, id: ShardId) -> Option<Arc<ShardMetadata>>;
 }
```

Comment:
How about removing all the trait parafernalia? With this PR we will only have one provider (the unbounded one) so maybe is a little soon for this generalisations that at the moment only add overhead. 
What do you think?